src/int.cr

```diff
@@ -237,6 +237,12 @@ struct Int
     end
   end
 
+  def <=>(other : Int)
```

Comment:
Maybe declare return type as `Int32` analog to my suggestion for `Number#<=>`. It also wouldn't hurt to copy the doc comment with a note that comparing ints won't return `nil`.

Reply:
Done